test/usecases/retrieveAverageVisitsPerDayByTrustId.test.js

```diff
@@ -53,14 +56,17 @@ describe.skip("retrieveAverageVisitsPerDay", () => {
     };
 
     const {
-      averageVisitsPerDay,
+      // TODO: Uncomment the following line once we understand fully why retrieveAverageVisitsPerDay returns a hardcoded value.
+      // averageVisitsPerDay,
       error,
     } = await retrieveAverageVisitsPerDayByTrustId(container)(
       trustId,
       new Date(2020, 6, 3)
     );
 
-    expect(averageVisitsPerDay).toEqual(2);
+    // TODO: Uncomment the following line once we understand fully why retrieveAverageVisitsPerDay returns a hardcoded value.
+    // expect(averageVisitsPerDay).toEqual(2);
```

Comment:
Looking at the original code, I think this is because the gateway being used to actually retrieve the number of average visits per day is just being mocked out and injected to the use case - so the test hardcodes the value so we can ensure that the use case is correctly interacting with the gateway.

It's not a hugely _valuable_ test in this case, it needs triangulation (maybe randomise the return value as a first pass?). But i think it can be added back in

Reply:
In this example, the container used for injection is ignored by the use case - so it would never use a real or a mocked out gateway implementation (src/usecases/retrieveAverageVisitsPerDayByTrustedId.js)

Reply:
So it is! My mistake, sorry